src/advect/AdvectorExplicitPredictorPatchOps.cpp

```diff
@@ -912,7 +912,7 @@ AdvectorExplicitPredictorPatchOps::getNumberCellGhosts() const
         return 3;
     case XSPPM7:
         return 4;
-    case UNKNOWN_LIMITER_TYPE:
+    default:
```

Comment:
@boyceg : Is it okay to put the default case here like this? We added some additional limiter_types that are not supported by this class. Keeping it as 'case UNKNOWN_LIMITER_TYPE' produced a ton of warnings.